README.md

```diff
@@ -1 +1,57 @@
 # drs-filer
+
+[![License][badge-license]][badge-url-license]
+[![Build_status][badge-build-status]][badge-url-build-status]
+[![Coverage][badge-coverage]][badge-url-coverage]
+[![GitHub_tag][badge-github-tag]][badge-url-github-tag]
```

Comment:
We don't have a tag yet, so this can be removed for now (and the corresponding links at the end of the document)